x-pack/plugins/security_solution/common/endpoint/index_data.ts

```diff
@@ -397,3 +402,46 @@ const indexFleetAgentForHost = async (
 
   return agentDoc;
 };
+
+const indexFleetActionsForHost = async (
+  esClient: Client,
+  endpointHost: HostMetadata
+): Promise<void> => {
+  const ES_INDEX_OPTIONS = { headers: { 'X-elastic-product-origin': 'fleet' } };
+  const agentId = endpointHost.elastic.agent.id;
+
+  for (let i = 0; i < 5; i++) {
+    // create an action
+    const isolateAction = fleetActionGenerator.generate({
+      data: { comment: 'data generator: this host is bad' },
+    });
+
+    isolateAction.agents = [agentId];
+
+    await esClient.index(
+      {
+        index: '.fleet-actions',
+        body: isolateAction,
+      },
+      ES_INDEX_OPTIONS
+    );
+
+    // Create an action response for the above
+    const unIsolateAction = fleetActionGenerator.generateResponse({
+      action_id: isolateAction.action_id,
+      agent_id: agentId,
+      action_data: {
+        command: isolateAction.data.command,
+        comment: 'data generator: bad host isolated',
```

Comment:
I _believe_ `action_data` === `isolateAction.data`, comment included. It's a parroted record of what the data payload from the action was.

**I think**

Reply:
Cool. I'll sync it up then. 👍